.github/workflows/ci.yml

```diff
@@ -16,38 +16,30 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rust_toolchain: [1.64.0, ""] # MSRV + ("" = rust-toolchain version)
+        rust_toolchain: [1.70.0, "stable"] # MSRV + ("" = rust-toolchain version)
```

Comment:
does the new action support 1.64.0? An msrv bump seems orthogonal, unless it's necessary

Reply:
> does the new action support 1.64.0? An msrv bump seems orthogonal, unless it's necessary

Running a `cargo update` breaks the build if not incremented.

Reply:
```suggestion
        rust_toolchain: [1.64.0, "stable"] 
```